nncf/config/schemata/experimental_schema.py

```diff
@@ -351,7 +361,7 @@
         "sparse_structure_by_scopes": {
             "type": "array",
             "items": MOVEMENT_SPARSE_STRUCTURE_BY_SCOPES_SCHEMA,
-            "description": "TBD"
+            "description": 'Describes how each supported layer will be sparsified.'
         },
         **SCOPING_PROPERTIES,
         **COMPRESSION_LR_MULTIPLIER_PROPERTY
```

Comment:
Please set up defaults for the schema in the similar manner as it is done in `nncf/config/schemata/defaults.py` - e.g. define a constant for each default value, make sure to reuse the same constants in your `config.get(...)` calls for default values, add the constant value as an attribute (`with_attributes(..., default=DEFAULT_VALUE)`), and remove the default value specification from the "description". Also make sure that the fields that are required are correctly specified via `"required"` as per JSON schema spec.